anndata/core/anndata.py

```diff
@@ -837,13 +494,15 @@ def _init_as_actual(
             self._raw = None
         elif isinstance(raw, Raw):
             self._raw = raw
-        else:
-            # is dictionary from reading the file, nothing that is meant for a user
+        elif isinstance(raw, cabc.Mapping):
+            # is dictionary from reading the file,
+            # nothing that is meant for a user
```

Comment:
Then it shouldn’t be here, but in the function reading from a file. We only want convenience-dict-initialization for things users should be able to supply.

Reply:
Addressed in df9fcc6